bcipy/signal/model/offline_analysis.py

```diff
@@ -311,8 +309,14 @@ def analyze_gaze(
         symbol_set=alphabet()
     )
 
-    # Extract the data for each target label and each eye separately.
     # Apply preprocessing:
+    inquiry_length = inquiries_list[0].shape[1]  # number of time samples in each inquiry
+    preprocessed_array = np.zeros((len(inquiries_list), 4, inquiry_length))
```

Comment:
4?

Reply:
It is a built-in size for the dimension of `gaze_data` (left eye x, left eye y, right eye x, right eye y). I will make it more pythonic